apps/todo/angular/src/app/todo/todo-footer/todo-footer.component.html

```diff
@@ -10,24 +10,14 @@
       </ng-template>
     </span>
   </p>
+    <div class="btn-group btn-group-horizontal lg:btn-group-horizontal p-4">
```

Comment:
keep the code as defined in the prettier config, otherwise every time we run nx format it will create commits on files that shouldn't be changed

Reply:
gocha